staging/src/k8s.io/kubectl/pkg/util/util.go

```diff
@@ -41,15 +39,6 @@ func ParseRFC3339(s string, nowFn func() metav1.Time) (metav1.Time, error) {
 	return metav1.Time{Time: t}, nil
 }
 
-// HashObject returns the hash of a Object hash by a Codec
```

Comment:
/cc @soltysh 

kubectl change x-ref https://github.com/kubernetes/kubernetes/issues/129652#issuecomment-3225042654

I can't find any evidence that this code is used https://github.com/kubernetes/kubernetes/issues/129652#issuecomment-3215980836

AFAICT it is stale / dead code.